contract/bridge/contracts/ExchangeRateOracle.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.12;
+
+import {ICollateral} from "./Collateral.sol";
+
+abstract contract ExchangeRateOracle is ICollateral {
```

Comment:
`ICollateral` records user and vaults collateral data, here may don't need to inherit it.

Reply:
Removed.